Library/Homebrew/dev-cmd/bottle.rb

```diff
@@ -75,6 +77,8 @@ def bottle_args
              description: "Specify a committer name and email in `git`'s standard author format."
       flag   "--root-url=",
              description: "Use the specified <URL> as the root of the bottle's URL instead of Homebrew's default."
+      flag   "--download-strategy=",
```

Comment:
```suggestion
      flag   "--root-url-download-strategy=",
```
or
```suggestion
      flag   "--root-url-using=",
```

Reply:
Ah, yeah, makes a lot of sense. Went with `--root-url-using` with the logic that it should be directly obvious to someone who is trying to get one of these commands to replicate a `using` clause they've probably already tried out by hand.